pkg/scaffold/project/tools.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018 The Operator-SDK Authors
```

Comment:
Using a `tools.go` file to force vendoring of un-imported tools is a [best practice](https://github.com/golang/go/issues/25922#issuecomment-402918061) for at least Go 1.11.

Reply:
```suggestion
// Copyright 2019 The Operator-SDK Authors
```